apps/api/.env.example

```
@@ -2,7 +2,7 @@
 NODE_ENV=development
 PORT=5800
 WS_PORT=5801
-ORIGIN=http://localhost:5700,http://localhost:5173
+ORIGIN=http://localhost:5173
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ORIGIN key should go before the PORT key

(UnorderedKey)